src/Lumi/Components/LabeledField.purs

```diff
@@ -19,6 +19,8 @@ data RequiredField
   | Optional
   | Neither
 
+derive instance eqRequiredField :: Eq RequiredField
```

Comment:
This is not being used in this PR now. It was used in earlier iterations of this same PR. Should I remove it?

Reply:
This seems like a sensible thing to have, especially since all the constructors are exported so you could reconstruct the same function anyway if you wanted to